website/content/docs/agent/template-config.mdx

```diff
@@ -0,0 +1,55 @@
+---
+layout: docs
+page_title: Vault Agent Template Config
+description: |-
+  Vault Agent's Template Config to set Templating Engine behavior
+---
+
+# Vault Agent Template Config
+
+Template Config allows users to determine the behavior that
+pertains to the templating engine within Vault Agent.
```

Comment:
> I referenced "templating engine" on purpose instead of template because the params within the template stanza defines the rendering behavior for templates themselves so I didn't want to confuse the two.

Oops, yeah I definitely missed that distinction! It's a tricky one to communicate well, because it's kind of spilling over into explaining the internals of Vault Agent rather than just explaining the behavior. WDYT of this suggestion? Possibly too simplistic, but I think it's complemented very well by the paragraph after it.

```suggestion
Template Config configures Vault Agent behavior common to all `template` stanzas.
```

Reply:
I've just noticed the "Functionality" section below as well, which goes into slightly more detail on the same subject. Perhaps worth merging these two sections together?

Reply:
This initial section is meant to be a 1-2 sentence general description of the feature. The functionality section goes a bit deeper into the details, and also has "Interactions with ..." sub-section.